vector-im/element-x-android
# Confirm back navigation when editing a poll only if the poll was changed

## Type of change


## Content

Confirm back navigation when editing a poll. If the content of the poll is changed, display a confirmation dialog.

## Motivation and context

- Part of https://github.com/vector-im/element-meta/issues/2179

## Screenshots / GIFs



## Tests

- Start editing a poll without changing anything
- Press back
- Observe no dialog
- Edit a poll and change something
- Press back
- Observe dialog

## Tested devices

- OS version(s): 13

## Checklist
